src/database/models/__init__.py

```diff
@@ -14,7 +14,15 @@ class Meta:
 
 
 class KhojUser(AbstractUser):
+    class SubscriptionType(models.TextChoices):
+        TRIAL = "trial"
+        STANDARD = "standard"
+
     uuid = models.UUIDField(models.UUIDField(default=uuid.uuid4, editable=False))
+    subscription_type = models.CharField(
```

Comment:
Do you think `Subscription` would better serve as a separate, isolated table? I imagine it would be safer to separate that from the core `User` object in order to keep it less cluttered over time. We can also use the `BaseModel` in that case. Open to hear your thoughts.

```python
class Subscription(BaseModel):
   class SubscriptionType(models.TextChoices):
      TRIAL = "trial"
      STANDARD = "standard"
   user = models.OneToOneField(KhojUser, on_delete=models.CASCADE)
   subscription_type = models.CharField(max_length=20, choices=SubscriptionType.choices, default=SubscriptionType.TRIAL)
   subscription_renewal_date = models.DateTimeField(null=True, default=None)
```